istresearch/rapidpro
# P4-3641: send sched msg failure fix

# For the Reviewer

When this is complete, you should approve the PR via github.

# For the Reviewee

## Summary
Fixed: Scheduling a Send Message to a Contact fails silently (no redirect to scheduler).

#### Release Note
Scheduling a message to send to contact(s) will correctly forward to the scheduler.

#### Breaking Changes
None.

## Quality Assurance

You have gathered the following items:

Given all of the items above, you have updated your Application ORT at the following locations:
- **Features and Alerting**: <link to app ORT>Required.
- **P4 Alerting**: <link to p4-alerting PR>Required.

## Testing and Verification

*Steps to test your application for someone not familiar with it.* Required.

Locally, or on the Dev system:
1. verify that the Page Source contains `<script>window.TEMBA_COMPONENTS_VERSION="0.13.3"</script>` and not "0.26.3". 
2. Send a contact a message, scheduling it for later. After the dialog closes, the page should forward to the scheduler correctly after a second (or two).

NOTE:  also fixed but hard to verify/test is that the "Are you sure?" popup now only ever happens once.  There have been complaints and witnessed accounts of seeing the "Are you sure?" multiple times, but since my refactoring of the JavaScript controlling that behavior, I've not witnessed it happen.  Please let me know if you witness multiple "Are you sure?" popups.
